build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.0'
+        classpath 'com.android.tools.build:gradle:4.0.1'
```

Comment:
Android Studio 4.1 now gives us a warning there:

https://github.com/rmst/yoke-android/blob/b51bed9ceacc9238c3ca935dae290f600ed92f6f/app/src/main/java/com/simonramstedt/yoke/YokeActivity.java#L620-L621

> Resource IDs will be non-final in Android Gradle Plugin version 5.0, avoid using them in switch case statements.

I guess that means we should change this switch to if.

Reply:
Does this help? f134e5f765644c4ed58c40b6eed334cbe15f0a42

Reply:
Looks good to me.